tests/integration/components/bs-form-test.js

```diff
@@ -1147,4 +1147,38 @@ module('Integration | Component | bs-form', function (hooks) {
     );
     assert.dom('input').doesNotHaveAttribute('readonly');
   });
+
+  /**
+   * @see https://github.com/kaliber5/ember-bootstrap/issues/1414
+   */
+  test('it does not have a showOwnValidation bug', async function (assert) {
+    assert.expect(1);
+    this.set('submit', ({ name }) => assert.equal(name, 'Zoey'));
+    this.set('beforeSubmit', (model) => {
+      if (this.user) {
+        model.set('name', this.user.name);
+      }
+    });
+
+    await render(hbs`
+      <BsForm
+        @model={{this}}
+        @onBefore={{this.beforeSubmit}}
+        @onSubmit={{this.submit}}
+        as |Form|
+      >
+        {{#unless this.user}}
```

Comment:
Is removing the form element the essential thing here in this test, that triggers the exception?

Reply:
In my scenario it is the combination of using `fillIn` on the element before then removing it that triggered the error.

I had another instance (not in tests) where the same thing was happening on a login form - the error was triggered when a route transition was started as part of a successful submit. In that case, my best guess was that the form being ripped out of the DOM as part of the transition seemed to trigger it.